src/main/kotlin/com/hibob/academy/resource/PetResource.kt

```diff
@@ -26,6 +26,13 @@ class PetResource(private val petService: PetService) {
         return Response.ok(petService.getPetById(id, companyId)).build()
     }
 
+    @GET
+    @Path("/{companyId}")
+    fun getPetByCompanyId(@PathParam("companyId") companyId:Long): Response {
+        return Response.ok(petService.getPetsByCompanyId(companyId)).build()
+    }
+
+
     @POST
     fun createPet(petCreationRequest: PetCreationRequest): Response {
```

Comment:
Were you requested to add @RequestBody? (I wasn't the person who gave this lecture so just want to make sure)